website/cue/reference/components/transforms/base/exclusive_route.cue

```diff
@@ -0,0 +1,29 @@
+package metadata
```

Comment:
Maybe we could update the docs for `route` too to point to `exclusive_route` (and vice versa) to make it easier for users to find which they want.